CHANGE-NOTES.html

```diff
@@ -1,11 +1,15 @@
 <ul>
-  <li>Added editor for .git/machete file with syntax highlighting</li>
-  <li>Added action for fast-forwarding parent to the given branch</li>
-  <li>Added action for resetting the given branch to its remote tracking branch</li>
-  <li>Added fetch/pull actions</li>
-  <li>Added help action</li>
-  <li>Improved displaying of status</li>
-  <li>Improved push action to automatically suggest force-push when needed</li>
-  <li>Improved stability of push dialog (v0.5.1)</li>
-  <li>Improved stability of slide out (v0.5.2)</li>
+  <li>Added support for pulling current branch</li>
+  <li>Added support for Slide-in new branch (and reattach existing one) without manual editing machete file</li>
+  <li>Open Machete Tab action was added under Git submenu with key shortcut Ctrl + Alt + Shift + M</li>
+  <li>Current branch is marked even if there is an ongoing operation in repo (rebasing, bisecting, etc.) and name of this operation is displayed</li>
+  <li>Upstream branches are inferred if not explicitly provided</li>
+  <li>Detecting already used indentation in machete file editor</li>
+  <li>Stdin and stdout are now included in error message when machete prerebase hook failed</li>
+  <li>Added notification in case when branch declared in machete file but nonexistent as local branch in repository is omitted</li>
+  <li>Refreshing of branch tree after slide-out action was fixed</li>
+  <li>Reset action is preceded by information dialog</li>
+  <li>Reset is prohibited when uncommitted changes are present in repository</li>
+  <li>Reset to outdated commit of remote branch is forbidden</li>
```

Comment:
Hmm not really `forbidden`... it's more about fixing a bug that sometimes caused the reset-to-remote action pick an outdated commit of the remote branch